backend/lib/models/Notification.js

```diff
@@ -0,0 +1,49 @@
+const { Schema, model } = require("mongoose");
+
+const notificationSchema = new Schema(
+  {
+    action: {
+      enum: ["like", "comment", "post"],
+      required: true,
+      type: String,
+    },
+    post: {
+      id: String,
+      title: String,
+    },
+    receiver: {
+      ref: "User",
+      required: true,
+      type: Schema.Types.ObjectId,
+    },
+    seenAt: Date,
+    sentAt: Date,
+    triggeredBy: {
+      id: String,
+      name: String,
+      photo: String,
+      type: String,
+    },
+  },
+  { collection: "notifications", timestamps: true },
+);
+
+/* eslint-disable sort-keys */
+notificationSchema.index({
+  createdAt: -1,
+  expireAfterSeconds: 1296000,  // 15 days
+});
```

Comment:
I am not sure but maybe this should be
```js
notificationSchema.index({
   createdAt: -1,
},
{ expireAfterSeconds: 1296000, }
);
```
As in [mongodb docs](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-after-a-specified-number-of-seconds)

Reply:
Yup, good catch.